### ISSUE TITLE
Fillet caught fish with knife

### ISSUE THREAD

**Issue by K4TFJ** [karkkant/valheim-spearfishing#3]
Enhancement request:  Add the ability for the player to fillet (craft) the caught Fish to its cookable component if the player's active weapon is a knife (flint, bronze, etc.) . This would be a great QOL improvement for early game which seems to blend quite well the spearfishing mod. I envision this occurring on the player's crafting GUI (no cauldron required).

**Comment by karkkant** [comment]
I agree that would be pretty cool feature. Sadly I'm not actively modding for Valheim atm. 
Perhaps I look into this if I find the time and motivation. 😃 

Luckily the cauldron is very early improvement so fish is easy to get on menu early on. Though fillet would be still be good if you need to cook outside the base for some reason.

**Comment by Goldenrevolver** [comment]
after using this mod, I equally ran into the issue of not getting raw fish anymore in the early game when fishing with a spear. and after finding this suggestion, I took the liberty to implement it in my own mod instead of turning it into a fork of spearfishing. that will make it easier to find when people search for it. also spearfishing works perfectly with the newest update, so there is no reason to upload a derivative version.

I also really liked your idea of spawning blood on hit using the deathsquito prefab because its so neutral (aka no body parts). I toyed around with it and found that the 'fx_TickBloodHit' effect from mistlands ticks worked really well too since its a bit smaller in radius, so that's the one I used for the raw fish cutting mod.

here is the thunderstore link: https://valheim.thunderstore.io/package/Goldenrevolver/Quickly_Cut_Raw_Fish_Without_Cauldron/
here is the nexus link: https://www.nexusmods.com/valheim/mods/2574
here is the github link: https://github.com/Goldenrevolver/ValheimMiniMods
